docs/source/es/quicktour.mdx

```diff
@@ -123,14 +123,21 @@ A continuación, carga el dataset (ve 🤗 Datasets [Quick Start](https://huggin
 >>> dataset = datasets.load_dataset("PolyAI/minds14", name="en-US", split="train")  # doctest: +IGNORE_RESULT
 ```
 
+Debemos asegurarnos de que la frecuencia de muestreo del conjunto de datos coincide con la frecuencia de muestreo con la que se entrenó `facebook/wav2vec2-base-960h`.
```

Comment:
@osanseviero could you take a look at my amazing Spanish skills here? haha

Reply:
Yes, your Spanish was flawless :rocket: nice!

Reply:
DeepL really is not bad then :sweat_smile: